models/staging/tpdm/base/base_tpdm__persons.sql

```diff
@@ -0,0 +1,20 @@
+with persons as (
+    {{ source_edfi3('persons') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?
```suggestion
),

renamed as (
```